.github/workflows/publish_docker_image.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow will run when changes are detected in the `main` branch, which
+# must include an update to the `docker/service_version.txt` file. The workflow
+# can also be manually triggered by a repository maintainer. This workflow will
```

Comment:
If this workflow can be triggered by a maintainer, but it doesn't have a change to the `service_version.txt` file, will that overwrite a release?

Reply:
I think that scenario would probably cause issues; for example part of the workflow is creating a git tag, and that would be a problem. So far these workflows have relied on people not trying to do manual releases. (But if you give someone a button, they will try and press it eventually)

The primary reason I added the manual release to these repositories was to enable the release of v1.0.0. I could definitely be convinced to remove the manual trigger in a subsequent PR.